myScripts.js

```diff
@@ -0,0 +1,107 @@
+//Download Form: Setting the Place of Practice Variable to "N/A"
+function setPlaceVariable() {
+    if (document.getElementById('downloadYes').checked) {
```

Comment:
you can just do a if statement to check if `downloadYes` is not checked by adding a `!` infront of a boolean (so `!true` will be `false`)

```
if (!document.getElementById('downloadYes').checked) {
   document.getElementById("downloadPlaceInput").value = "N/A";
}
```